GroceryApp/GroceryApp/GroceryTrip/FeedbackView.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  FeedbackView.swift
+//  GroceryApp
+//
+//  Created by AGGARWAL, CHETNA [AG/1000] on 4/29/19.
+//
+
+import Foundation
+import UIKit
+
+class FeedbackView: UIView {
+    
+    @IBOutlet weak var feedBack: UILabel!
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        initSelf()
+    }
+    
+    override func awakeFromNib() {
+        feedBack.text = ""
+        feedBack.backgroundColor = UIColor.white
+        feedBack.layer.borderWidth = 1
+        feedBack.layer.borderColor = UIColor.darkGray.cgColor
+        feedBack.layer.cornerRadius = bounds.width/1.8
+    }
+    
+    func updateFeedBackText(message: String) {
+        feedBack.text = message
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        initSelf()
```

Comment:
what does initSelf do again??

Reply:
initSelf is setting some default properties.